doc/guide/packages.xml

```diff
@@ -186,6 +186,12 @@ $ cockpit-bridge --packages
         <listitem><para>The relative path to the HTML file within the package that implements
           the menu item or tool.</para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>docs</term>
+        <listitem><para>This item contain list of documentation for the given page.
+          Each item is object containing <code>name</code> and <code>url</code>.
```

Comment:
is "an" object. "name" is the menu entry, right? perhaps s/name/label/, or add a sentence like ", where "name" is what will appear in the Help menu."